04_fep/05_ATM/01-ATM.ipynb

```diff
@@ -0,0 +1,481 @@
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ca = [atom.index() for atom in solvated.search(\"atomname CA\")]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4ab9036b-5cd8-40ee-b1e4-38dff16eb9d4",
+   "metadata": {},
+   "source": [
+    "Now we can create a minimisation protocol using the forces and `atm_data` dictionary we created when setting up the system."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "e1de322e-f178-444e-bfc7-8f5936182b96",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "minimisation = BSS.Protocol.ATMMinimisation(\n",
+    "    data=atm_data, core_alignment=True, restraint=ca, com_distance_restraint=True\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "56b04fa5-aa1e-4032-9ca6-e79a21240845",
+   "metadata": {},
+   "source": [
+    "This protocol can then be run using the BioSimSpace openMM runner."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "03f63303-98be-48b0-ba1a-3c82653a4c30",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "minimisation_process = BSS.Process.OpenMM(solvated, minimisation)\n",
+    "minimisation_process.start()\n",
+    "minimisation_process.wait()\n",
+    "minimised = minimisation_process.getSystem(block=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "31954b81-0307-42d3-8c4c-489a62be74c1",
+   "metadata": {},
+   "source": [
+    "We will now run our first equilibration, including all of the forces we used in the minimisation. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "6760d38d-bc66-49a1-b909-395f798d9d5e",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "equilibration = BSS.Protocol.ATMEquilibration(\n",
+    "    data=atm_data,\n",
+    "    core_alignment=True,\n",
+    "    restraint=ca,\n",
+    "    com_distance_restraint=True,\n",
+    "    runtime=\"100ps\",\n",
+    ")\n",
+    "equilibrate_process = BSS.Process.OpenMM(minimised, equilibration)\n",
+    "equilibrate_process.start()\n",
+    "equilibrate_process.wait()\n",
+    "equilibrated = equilibrate_process.getSystem(block=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9d2da0d7-f7ee-45de-9af5-a54b2f03ba4c",
+   "metadata": {},
+   "source": [
+    "The system has now been minimised and equilibrated as a vanilla system, that is without the ATMForce present. \n",
+    "\n",
+    "We now need to introduce the ATMForce to the system; this introduction needs to be gradual, otherwise our simulations will crash.\n",
+    "\n",
+    "This gradual introduction can be done by annealing the system to a lambda value of 0.5. Starting from lambda=0, the annealing protocol simulates a series of windows in which the value of lambda is gradually increased. In this case we will use 10 annealing cycles with a total runtime of 100ps, meaning that the value of lambda is increased by a value of 0.05 every 10ps."
```

Comment:
same comment as for equilibration (attempt to run with a shorter runtime for the purpose of the demo)

Reply:
Now cut to 1ps

Reply:
Update the comment cell  to add 

For production runs we recommend extending the runtime to at least 100 ps.